api/src/resolvers/campaigns/swollen-sunday/swollen-sunday-campaign-cypher.ts

```diff
@@ -64,7 +64,7 @@ CALL apoc.cypher.run('
  RETURN DISTINCT bussing  LIMIT 4',
  {bacenta:bacenta}) YIELD value
 
-WITH avg(value.bussing.attendance) as averageBacentaAttendance, bacenta
+WITH ROUND(avg(value.bussing.attendance)) as averageBacentaAttendance, bacenta
```

Comment:
Why are you introducing this back?

Reply:
Rounding before the end of the data is what brings the large errors in target distribution or?

Reply:
And also for Line 79, I thought you were going to change that to the bacenta.id so that it is something more lasting. 

We may one day delete these bacenta codes, but we are not likely to delete the bacenta ids

Reply:
> Rounding before the end of the data is what brings the large errors in target distribution or?

Yes, however in the last test I did, I got values less than the overall target I put in. I feel it's better to go over then under.